Library/Homebrew/diagnostic.rb

```diff
@@ -113,10 +113,15 @@ def build_error_checks
 
       def please_create_pull_requests(what = "unsupported configuration")
         <<~EOS
-          You will encounter build failures with some formulae.
-          Please create pull requests instead of asking for help on Homebrew's GitHub,
-          Twitter or any other official channels. You are responsible for resolving
-          any issues you experience while you are running this
+          It is expected behaviour that some formulae will fail to build.
+          It is expected behaviour that Homebrew will be buggy and slow.
+          Do not create any issues about this on Homebrew's GitHub repositories.
+          Do not create any issues even if you this message is unrelated.
+          Any opened issues will be immediately closed without response.
+          Do not ask for help from MacHomebrew on Twitter.
+          You may ask for help in Homebrew's discussions but are likely to receive no response.
+          Try to figure out the problem yourself and submit a fix as a pull request.
+          We will review it but may or may not accept it.
           #{what}.
```

Comment:
We probably want to keep the `You are responsible...` part, or drop this `#{what}`, since there will just be a random
```
unsupported configuration
```
at the end of the message here.